bclientapi/bclientapi.go

```diff
@@ -48,7 +49,7 @@ func addFileToTransactionList(filename string, fileID string, item string, isNew
 	fileIDMutex.Unlock()
 }
 
-func New(server string, item string, fileroot string, chunkSize int, wait bool) *itemAttributes {
+func New(server string, item string, fileroot string, chunkSize int, wait bool, token string) *itemAttributes {
```

Comment:
I do prefer that style since at some point New() will have an absurd number of parameters.

To each their own.